daemon/protocol/python/setup.py

```diff
@@ -1,12 +1,14 @@
 #!/usr/bin/env python
 
 from setuptools import setup
+from SystemEvents.Folder_Actions_Suite import scripts
 
 setup(
   name = 'Scoot',
   version = '0.5',
   description = 'Scoot library for connecting to a local Scoot Daemon Server',
   url = 'https://github.com/scootdev/scoot',
   packages = ['scoot'],
-  install_requires = ['grpcio']
+  install_requires = ['grpcio', 'docopt'],
```

Comment:
Might be a good time to add a requirements.txt with these also and have our dependency instructions be: "pip install -r requirements.txt"
@JieGhost - thoughts?